docs/live-migration.md

```diff
@@ -0,0 +1,107 @@
+Live migration is the act of moving a virtual machine (VM) from one
+hypervisor to another while maintaining the connectivity and liveness
+of the applications running on the virtual machine.  It is one of the
+primary advantages of using an external VM management service.
+
+Triggering a live migration requires that the user perform an API
+call.
+
+To Create (start migration):
+
+```bash
+
+kubectl create -f migration.yaml
+```
+
+Here is an example of a  migration request in YAML format:
+
+```yaml
+
+apiVersion: kubevirt.io/v1alpha1
+kind: Migration
+metadata:
+  name: testvm-migration
+spec:
+  selector:
+    name: testvm
+```
+
+
+The selector section indicates which VM objects are to be
+migrated. The name field shown here would match a VM with the `name`
+value of `testvm.`
+
+
+To list all migrations:
+
+```bash
+
+kubectl get migrations
+```
+
+To query a specific migration named `testvm_migration`:
+
+```bash
+
+kubectl get migrations testvm_migration -o yaml
+```
+
+Which produces:
+
+```yaml
+
+apiVersion: kubevirt.io/v1alpha1
+kind: Migration
+metadata:
+  creationTimestamp: 2017-03-15T11:25:47Z
+  name: testvm-migration
+  namespace: default
+  resourceVersion: "96512"
+  selfLink: /apis/kubevirt.io/v1alpha1/namespaces/default/migrations/testvm-migration
+  uid: 230cb349-0972-11e7-b9cf-525400b9ab10
+spec:
+  selector:
+    name: testvm
+status:
+  phase: Succeeded
+
+```
+
+To cancel `testvm_migration`:
+
+```bash
+
+kubectl delete migrations testvm_migration 
```

Comment:
This line is misleading because the migration has a random suffix which is not acknowledged here.